airbyte-integrations/connectors/source-trello/source_trello/source.py

```diff
@@ -0,0 +1,227 @@
+            params["since"] = stream_state[self.cursor_field]
+        return params
+
+    def get_updated_state(self, current_stream_state: MutableMapping[str, Any], latest_record: Mapping[str, Any]) -> Mapping[str, Any]:
+        return {self.cursor_field: max(latest_record.get(self.cursor_field, ""), current_stream_state.get(self.cursor_field, ""))}
+
+
+class Boards(TrelloStream):
+    """Return list of all boards.
+    API Docs: https://developers.intercom.com/intercom-api-reference/reference#list-attached-segments-1
+    Endpoint: https://api.trello.com/1/members/me/boards
+    """
+
+    def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
+        return "members/me/boards"
+
+
+class Cards(ChildStreamMixin, TrelloStream):
+    """Return list of all cards of a boards.
+    API Docs: https://developer.atlassian.com/cloud/trello/rest/api-group-boards/#api-boards-id-cards-get
+    Endpoint: https://api.trello.com/1/boards/<id>/cards/all
+    """
+
+    parent_stream_class = Boards
+    limit = 20000
+    extra_params = {"customFieldItems": "true"}
+
+    def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
+        return f"boards/{stream_slice['id']}/cards/all"
+
+
+class Checklists(ChildStreamMixin, TrelloStream):
+    """Return list of all checklists of a boards.
+    API Docs: https://developer.atlassian.com/cloud/trello/rest/api-group-boards/#api-boards-id-checklists-get
+    Endpoint: https://api.trello.com/1/boards/<id>/checklists
+    """
+
+    parent_stream_class = Boards
+    extra_params = {"fields": "all", "checkItem_fields": "all"}
+
+    def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
+        return f"boards/{stream_slice['id']}/checklists"
+
+
+class Lists(ChildStreamMixin, TrelloStream):
+    """Return list of all lists of a boards.
+    API Docs: https://developer.atlassian.com/cloud/trello/rest/api-group-boards/#api-boards-id-lists-get
+    Endpoint: https://api.trello.com/1/boards/<id>/lists
+    """
+
+    parent_stream_class = Boards
+
+    def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
+        return f"boards/{stream_slice['id']}/lists"
+
+
+class Users(ChildStreamMixin, TrelloStream):
+    """Return list of all members of a boards.
+    API Docs: https://developer.atlassian.com/cloud/trello/rest/api-group-boards/#api-boards-id-members-get
+    Endpoint: https://api.trello.com/1/boards/<id>/members
+    """
+
+    parent_stream_class = Boards
+
+    def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
+        return f"boards/{stream_slice['id']}/members"
+
+
+class Actions(ChildStreamMixin, IncrementalTrelloStream):
+    """Return list of all actions of a boards.
+    API Docs: https://developer.atlassian.com/cloud/trello/rest/api-group-boards/#api-boards-boardid-actions-get
+    Endpoint: https://api.trello.com/1/boards/<id>/actions
+    """
+
+    parent_stream_class = Boards
+    limit = 1000
```

Comment:
parent_stream_class has the single value 'Boards'. And it makes no sense to separate it.

Reply:
parent_stream_class only now has value `Boards` in future if we need to add new streams better to use more variable class. So, it have sense to separate it.

Reply:
Ok, but this value is defined several times. Maybe will we do it as default value?

Reply:
I think that will be matched with the abstract logic of ChildStreamMixin.